src/meson.build

```diff
@@ -9,7 +9,7 @@ extdir = with_webext_dir[0]
 
 gresources_dir = include_directories('gresource')
 gnome = import('gnome')
-utils = '@0@/utils.sh'.format(meson.build_root())
+utils = join_paths(meson.source_root(), 'build/utils.sh')
```

Comment:
Which version of meson was `join_paths()` introduced. We need to stay compatible with the meson that is available on Debian currently.

Reply:
http://mesonbuild.com/Reference-manual.html#join_paths says "Added 0.36.0"

Reply:
Ah okay then we're good :+1:  Thanks!